src/sentry/testutils/pytest/fixtures.py

```diff
@@ -362,18 +378,15 @@ def inner(output, reference_file=None, subname=None):
             )
 
         try:
-            with open(reference_file, encoding="utf-8") as f:
-                match = _yaml_snap_re.match(f.read())
-                if match is None:
-                    raise OSError()
-                _header, refval = match.groups()
+            _, refval = read_snapshot_file(reference_file)
         except OSError:
             refval = ""
 
         refval = refval.rstrip()
         output = output.rstrip()
+        is_unequal = inequality_comparator(refval, output)
```

Comment:
Is `is_unequal` accurate here?

Seeing from above
```
inequality_comparator=lambda refval, output: refval == output,
```

Reply:
Ah good catch, I reversed these (changed is_equal -> is_unequal) but forgot to flip the base case.

Reply:
Re-uploaded, with this fixed.